krm/1_clusters/cluster-dev.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: container.cnrm.cloud.google.com/v1beta1
+kind: ContainerCluster
+metadata:
+  name: dev-us-central1
+  namespace: YOUR_NAMESPACE # {"$kpt-set":"namespace"}
+  annotations:
+    cnrm.cloud.google.com/remove-default-node-pool: "true" # don't create the default node pool
+spec:
+  initialNodeCount: 1 # this field must be set to greater than 0
+  addonsConfig:
+    horizontalPodAutoscaling:
+      disabled: false
+    httpLoadBalancing:
+      disabled: false
+    istioConfig:
+      disabled: true
+    networkPolicyConfig:
+      disabled: false
+  clusterAutoscaling:
+    enabled: false
+  databaseEncryption:
+    state: DECRYPTED
+  defaultMaxPodsPerNode: 110
```

Comment:
Do we need to set all these? If we're just repeating defaults we shouldn't include them in the KRM.